cmd/feederd/main.go

```diff
@@ -1,5 +1,96 @@
+// Copyright (c) 2020 The VulpemVentures developers
+
+// Feeder allows to connect an external price feed to the TDex Daemon to determine the current market price.
 package main
 
+import (
+	"flag"
+	"os"
+	"os/signal"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/gorilla/websocket"
+	"github.com/tdex-network/tdex-feeder/config"
+	"github.com/tdex-network/tdex-feeder/pkg/conn"
+	"github.com/tdex-network/tdex-feeder/pkg/marketinfo"
+
+	pboperator "github.com/tdex-network/tdex-protobuf/generated/go/operator"
+)
+
+const (
+	defaultConfigPath = "./config.json"
+)
+
 func main() {
```

Comment:
The main function should be shorter: I would separate this function into several parts: `setup` part and `infinite loop` for instance